docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -52,38 +52,81 @@ The LogPipeline resource is managed by the Telemetry Operator, a typical Kuberne
 
 The Telemetry Operator watches all LogPipeline resources and related Secrets. Whenever the configuration changes, it validates the configuration (with a [validating webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)) and generates a new configuration for the Fluent Bit DaemonSet, where several ConfigMaps for the different aspects of the configuration are generated. Furthermore, referenced Secrets are copied into one Secret that is mounted to the DaemonSet as well.
 
-## Set up the LogPipeline
-
-### Step 1: Create a LogPipeline
-
-1. To ship application logs to a new output, create a resource file of kind LogPipeline:
-
-   ```yaml
-   kind: LogPipeline
-   apiVersion: telemetry.kyma-project.io/v1alpha1
-   metadata:
-     name: http-backend
-   spec:
-     output:
-      http:
-        dedot: false
-        format: "json"
-        host:
-          valueFrom:
-            secretKeyRef:
-              name: my-secret
-              namespace: my-namespace
-              key: host
-        port: "80"
-        uri: "/"
-        user:
-          value: "user"
-        password:
-          value: "not-required"
-        tls:
-          disabled: true
-          skipCertificateValidation: true
-   ```
+## Setting up a LogPipeline
+
+### Step 1: Create an Input
```

Comment:
```suggestion
### Step 1: Create an input
```
Our guidelines want [sentence case](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology/#capitalization) in headlines.